### PR TITLE
Add rotateLabels for discreteBarChart

### LINKED ISSUE #359: rotateLabels for discreteBarChart?
the rotateLabels() function seems to have no effect on discreteBarCharts. Is there any way to rotate the labels on this type of chart?

### PR DESCRIPTION
**Pull request by dbabaioff**
This PR fixes #359

### CODE REVIEW

File: src/models/discreteBarChart.js [hunk 1]
```diff
@@ -141,6 +142,12 @@ nv.models.discreteBarChart = function() {
                         .selectAll('text')
                         .attr('transform', function(d,i,j) { return 'translate(0,' + (j % 2 == 0 ? '5' : '17') + ')' })
                 }
+                
+                if (rotateLabels)
+                    xTicks
+                        .selectAll('.tick text')
+                        .attr('transform', 'rotate(' + rotateLabels + ' 0,0)')
+                        .style('text-anchor', rotateLabels > 0 ? 'start' : 'end');
```
Comment (liquidpele) [inline src/models/discreteBarChart.js]: Please add brackets around all if blocks.

**Comment by liquidpele** [comment]
Also please add the new option to the json structure at the end of documentation.html :)

**Comment by dbabaioff** [comment]
It's already there: https://github.com/novus/nvd3/blob/master/examples/documentation.html#L734

**Comment by liquidpele** [comment]
Sweet, thanks!

**Pull request merged**